foundry.toml

```diff
@@ -9,6 +9,17 @@ block_number = 14126430
 block_timestamp = 1643802347
 runs = 4096
 libs = ["node_modules", "lib"]
+build_info = true
+extra_output = ['storageLayout']
```

Comment:
@xBA5ED can we add a small comment for these new values explaining what they do?